### DISCUSSION TITLE
How can I disable the files or show them lower in the search?

### DISCUSSION THREAD

**Discussion by justalemon** [JakubAndrysek/MkDoxy#82] [General]
Hi! I'm currently having a slight issue with MkDoxy where the source files are shown higher than the documentation itself:

![image](https://github.com/JakubAndrysek/MkDoxy/assets/11861253/db2ca40a-bbf0-4bdb-96b5-2e14e08ee1a2)

I was wondering how I can either:

- Disable the files
- Make the files show lower in the search

Thanks!

PS: This also happens on the MkDoxy documentation :)

![image](https://github.com/JakubAndrysek/MkDoxy/assets/11861253/e43ce000-d487-4c54-a7a2-bceefc45b2c3)

**Comment by JakubAndrysek** [comment]
Related conversation: https://github.com/JakubAndrysek/MkDoxy/issues/101#issuecomment-2065482023
